data_structures/linked_list/doubly_linked_list.py

```diff
@@ -3,7 +3,8 @@
 - This is an example of a double ended, doubly linked list.
 - Each link references the next link and the previous one.
 - A Doubly Linked List (DLL) contains an extra pointer, typically called previous pointer, together with next pointer and data which are there in singly linked list.
- - Advantages over SLL - IT can be traversed in both forward and backward direction.,Delete operation is more efficent"""
+ - Advantages over SLL - IT can be traversed in both forward and backward direction.,Delete operation is more efficent
+"""
```

Comment:
Can we please wrap these lines so that they can be read in the GitHub editor without left-right scrolling?

Reply:
@cclauss , Sure, I'll do it.